code/src/App.js

```diff
@@ -1,12 +1,32 @@
-import React from 'react';
-import data from './data.json';
+import React from 'react'
+import './index.css';
+import Artists from 'Components/Artists'
+import Header from 'Components/Header'
+/* import Images from 'Components/Images' */
+import Album from 'Components/Album';
+import data from './data.json'
 
 console.log(data);
-
 export const App = () => {
+  const imageIndex = 0; // index of the image you want to display
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <>
+      <Header />
+      <main className="parent-container">
+        <div className="album-container">
+          {data.albums.items.map((album) => (
+            <div key={album.id}>
+              <div className="Images">
+                <img src={album.images[imageIndex].url} alt={album.name} />
+              </div>
+              <a href={album.external_urls.spotify}>
```

Comment:
So, it looks like if you click on the album or the artist name, the link only goes to the album.  How could you change the links so that clicking on the artist takes you to the artist page?